src/main/kotlin/NoteArchive.kt

```diff
@@ -0,0 +1,66 @@
+package main.kotlin
+import java.util.Scanner
+class NoteArchive(val name: String){
+    private val scanner = Scanner(System.`in`)
+    private val notes = mutableListOf<Note>()
+
+        fun startMenu() {
+            while (true) {
+                println("1. Выбор заметки")
+                println("2. Создание заметки")
+                println("3. Вернуться к выбору архива")
+
+if (scanner.hasNextInt()) {
+                when (scanner.nextInt()) {
+                    1 -> selectNotes()
+                    2 -> createNotes()
+                    3 -> return
+                    else -> println("Неверный ввод. Пожалуйста, выберите опцию 1, 2 или 3.")
+                }
+            } else {
+    println("Неверный ввод. Пожалуйста, выберите опцию 1, 2 или 3 в цифровом выражении.")
+    scanner.next()
+}
+            }
+        }
+    private fun selectNotes(){
+        println("Выберите заметку:")
+
+        for ((index, note) in notes.withIndex()){
+            println("${index + 1}. ${note.title}")
+        }
+        val noteIndex = scanner.nextInt() - 1
+        if (noteIndex in 0 until notes.size) {
```

Comment:
```suggestion
        if (noteIndex in notes.indices) {
```
Используя оператор `until` создаётся объект `IntRange` (это просто к сведению), использование `indices` просто уберёт создание объекта в функцию доступа к проперти `indices` и так выглядит лучше, т.к. меньше похоже на цикл